packages/block-library/src/table/editor.scss

```diff
@@ -41,9 +41,19 @@
 
 	td.is-selected,
 	th.is-selected {
-		border-color: $blue-medium-500;
-		box-shadow: inset 0 0 0 1px $blue-medium-500;
-		border-style: double;
+		position: relative;
```

Comment:
What is the reason behind these changes? What does it have to do with alignment?

Reply:
It is a better way to apply a cell border than box-shadow, but not needed for this PR so I've removed these changes.